EmailSender/src/main/resources/hotelServiceURLs.properties

```diff
@@ -0,0 +1,2 @@
+url.get_user_by_id=http://localhost:8080/user/get/
```

Comment:
I think we can change it to  hotel-service.url==http://localhost:8080.
And in code use hotelServiceProperties.getUrl() + "/user/get"